netbox/wireless/filtersets.py

```diff
@@ -25,6 +25,17 @@ class WirelessLANGroupFilterSet(OrganizationalModelFilterSet):
         queryset=WirelessLANGroup.objects.all(),
         to_field_name='slug'
     )
+    ancestor_id = TreeNodeMultipleChoiceFilter(
```

Comment:
All of these in this file look like they are missing the label definition, don't know if you want to address it here or in a separate PR as fields beyond just PR are missing them.

Reply:
Not in scope for #15383; it'll need to be addressed separately (probably under #15237).